plugins/tutor-contrib-library-authoring-mfe/README.rst

```diff
@@ -0,0 +1,46 @@
+library_authoring_mfe plugin for `Tutor <https://docs.tutor.overhang.io>`_
+===================================================================================
+
+Installation
+------------
+
+Follow these instructions to enable this microfrontend:
+
+* Install `tutor nightly <https://github.com/overhangio/tutor/tree/nightly>`_: ``pip install -e 'git+https://github.com/overhangio/tutor.git@nightly#egg=tutor'``
+* Install `tutor-mfe nightly <https://github.com/overhangio/tutor-mfe/tree/nightly>`_: ``pip install -e 'git+https://github.com/overhangio/tutor-mfe.git@nightly#egg=tutor-mfe'``
+* To use blockstore with `minio <https://min.io/>`_
+  
+  * Install `tutor-minio <https://github.com/overhangio/tutor-minio>`_ nightly ``pip install -e 'git+https://github.com/overhangio/tutor-minio.git@nightly#egg=tutor-minio'``
+  * Enable minio plugin: ``tutor plugins enable minio``
+  * Enable the blockstore_config_minio plugin: ``tutor plugins enable blockstore_config_minio``
+
+* To use blockstore with django :code:`FileSystemStorage`
+
+  * Enable the blockstore_config_filesystem plugin: ``tutor plugins enable blockstore_config_filesystem``
+
+* Enable this plugin: ``tutor plugins enable library_authoring_mfe``
+* Save the tutor config: ``tutor config save``
+* Build mfe image: ``tutor images build mfe`` (if you have trouble here you may need to run it with ``--no-cache``) 
+* Launch tutor: ``tutor local launch``
+
+If you want to run this MFE in
+`development mode <https://github.com/overhangio/tutor-mfe/#mfe-development>`_
+(to make changes to the code), instead of step 9 above, do this::
+
+   cd /path/to/frontend-app-library-authoring
+   tutor dev run --mount=. library-authoring npm install  # Ensure NPM requirements are installed into your fork.
+   tutor dev start --mount=. library-authoring
```

Comment:
This changed as of 16.0.0.

```suggestion
   tutor config save --append MOUNTS=./frontend-app-library-authoring
   cd frontend-app-library-authoring
   nvm use && npm install
   tutor dev launch
```

(I'm unsure if `tutor dev run` would be better, here.  Haven't tested if it still works - but the local npm install seems to work.)